server/views/pages/barcode/pdf/print-coversheets.njk

```diff
@@ -61,9 +61,7 @@
     </div>
 
     {% if smokeTestBarcode %}
-    <div class="govuk-visually-hidden">
-      <p data-qa='smoke-test-barcode'>{{ smokeTestBarcode }}</p>
-    </div>
+      <span class="govuk-!-display-none" data-qa="smoke-test-barcode">{{ smokeTestBarcode }}</span>
```

Comment:
`govuk-!-display-none` is a better css class to use for this use case, and no need to wrap it in a div - we just need a simple element that cypress can target to get the value